packages/fiori-elements-writer/src/types.ts

```diff
@@ -103,14 +104,17 @@ export interface FioriApp extends App {
 
 export interface FioriElementsApp<T> extends Ui5App {
     template: Template<T>;
-    service: Omit<OdataService, 'model'>; // Model name will use defaults
+    service: Omit<OdataService, 'model'> & {
+        capService?: CapServiceCdsInfo;
+    };
     app: FioriApp;
     appOptions: Partial<AppOptions> & {
         /**
          * Generate OPA based tests, if applicable to the specified template.
          * This will eventually move up to {@link Ui5App.appOptions}
          */
         addTests?: boolean;
+        enableNPMWorkspaces?: boolean;
```

Comment:
Could you please add a detailed description of how this works as you cannot `enabledNPMWorkspaces` on an app so its confusing. Please also note thats this is dependant on the service being of type `capService`.

Reply:
Thanks @kjose90